workflow-manager/bucket/bucket.go

```diff
@@ -0,0 +1,161 @@
+package bucket
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	aws_session "github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/letsencrypt/prio-server/workflow-manager/tokenfetcher"
+	"google.golang.org/api/iterator"
+)
+
+type bucket struct {
```

Comment:
nit: Should this be `Bucket` so it's visible outside of this module?

Reply:
I didn't see anything that needs to see the actual bucket struct outside of the bucket module, so I went with the principle of data hiding.

I think if we end up needing the Bucket outside of this module then we can make it visible?

Reply:
Eh actually since we're using the Bucket struct outside it feels odd to keep it unexported. I'll make it exported.